scripts/integration-tests/e2e-prettier.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+#==============================================================================#
+#                                  SETUP                                       #
+#==============================================================================#
+
+# Start in scripts/integration-tests/ even if run from root directory
+cd "$(dirname "$0")" || exit
+root="$PWD"
+
+source utils/local-registry.sh
+source utils/cleanup.sh
+
+# Echo every command being executed
+set -x
+
+# Clone prettier
+git clone --depth=1 https://github.com/prettier/prettier /tmp/prettier
+cd /tmp/prettier || exit
+
+# Update @babel/* dependencies
+bump_deps="$root/utils/bump-babel-dependencies.js"
+node "$bump_deps"
+for d in ./packages/*/
+do
+  (cd "$d"; node "$bump_deps")
+done
+
+#==============================================================================#
+#                                 ENVIRONMENT                                  #
+#==============================================================================#
+node -v
+yarn --version
+
+#==============================================================================#
+#                                   TEST                                       #
+#==============================================================================#
+
+startLocalRegistry "$root"/verdaccio-config.yml
+yarn install
+
+if [ "$BABEL_8_BREAKING" = true ] ; then
+  # This option is removed in Babel 8
+  sed -i 's/allowDeclareFields: true,\?/\/* allowDeclareFields: true *\//g' babel.config.js
+fi
+
+# Only run js and jsx format tests
+yarn test "tests/format/jsx?"
```

Comment:
Should we add `--update-snapshot` here? I will expect the snapshot need updated when an error message is modified. Will `--update-snapshot` fail when a valid program throws or versa?

Reply:
I'm afraid  `--update-snapshot` can't be used here. Let me see if we can add an env flag to skip snapshot error messages, but only assert error throws.

Reply:
I tried to use different assertion, but it doesn't work. I guess we'll have to use `--update-snapshot`.

@sosukesuzuki WDYT?

Reply:
I think we have to use `--update-snapshot`.